hashira-web/functions/Makefile

```
@@ -1,5 +1,32 @@
 test:
 	go test ./...
 
+deploy-all:
+	make deploy-call
+	make deploy-ping
+	make deploy-upload
+	make deploy-download
+	make deploy-test-access-token
+
 deploy-call:
 	gcloud functions deploy call --entry-point Call --runtime go120 --memory 256MB --trigger-http --allow-unauthenticated --region asia-northeast1 --max-instances 10
+
+# deprecated
+deploy-ping:
+	gcloud functions deploy ping --entry-point Ping --runtime go116 --memory 256MB --trigger-http --allow-unauthenticated --region asia-northeast1 --max-instances 10
```

```suggestion
	gcloud functions deploy ping --entry-point Ping --runtime go120 --memory 256MB --trigger-http --allow-unauthenticated --region asia-northeast1 --max-instances 10
```

I'll update to go 1.20 after revert

---

https://github.com/pankona/hashira/commit/ee3e89b9e1660c737f8436b11ed01c9b904e0981